ldk/go/networkClient.go

```diff
@@ -12,17 +13,19 @@ type NetworkClient struct {
 	session *Session
 }
 
-// HTTPResponse is the structure recieved from HttpRequest
+// HTTPResponse is the structure received from HttpRequest
 type HTTPResponse struct {
 	ResponseCode int
 	Data         []byte
+	Headers      map[string]*structpb.ListValue
```

Comment:
We shouldn't return a protobuf type to the developer. Can we call `*structpb.ListValue.String()` to get a string and then return a map[string]string? 

Why are we using a `*structpb.ListValue` at all? Couldn't we just send a map of strings back the same as we send?

Reply:
Ok, I see the other conversation now. I agree that `map[string][]string` is a mess. But I think we should return the same format that the standard http method returns.

Reply:
"standard http method returns" implies going back to using `http.Header` and marshalling the data. Is that what you're saying here?

Reply:
@BillyMurrayOlive ^

Reply:
I tried doing `map[string][]string` but it really needs to be more like `map[string][]*proto.someRandomListDefinitionInProto`, and I've tried a couple different ways and I can seem to access these proto generated structs in sidekick.

Reply:
Does someone know a little more about that and can point me to an example?